pandas/tests/test_window.py

```diff
@@ -520,6 +520,26 @@ def test_closed(self):
         with pytest.raises(ValueError):
             df.rolling(window=3, closed='neither')
 
+    @pytest.mark.parametrize("func", [('min'), ('max')])
```

Comment:
you don't need the parens around 'min', 'max'

Reply:
Hi @jreback, thanks for pointing this out! Got it fixed.